tendenci/apps/events/models.py

```diff
@@ -1607,11 +1607,12 @@ def register_child_events(self, child_event_pks, is_admin=False):
                 child_event__repeat_uuid=event.repeat_uuid
             ).exclude(child_event_id=event.pk).exists():
 
-                current_event = self.registrantchildevent_set.filter(
+                # This is the orignal event with a matching repeat_uuid that the user is registered for.
+                original_event = self.registrantchildevent_set.filter(
                     child_event__repeat_uuid=event.repeat_uuid).first().child_event
                 error = _(
                     f'{event.title} on {event.start_dt.date()} is a repeat of event on ' \
-                    f'{current_event.start_dt.date()}. Please select only one.')
+                    f'{original_event.start_dt.date()}. Please select only one.')
                 raise Exception(error)
```

Comment:
original_event is closer in meaning to what we're doing here... it is the original event the user registered for that is a copy of an event the user is trying to register for